Makefile

```diff
@@ -150,3 +150,8 @@ merge_pr: ## Merge a PR. Must specify number in a PR=<number> form.
 find_diverged_forks: ## Check all github forks belonging to paas to see if they've diverged upstream
 	$(if ${GITHUB_TOKEN},,$(error Must pass GITHUB_TOKEN=<personal github token>))
 	./scripts/find_diverged_forks.py alphagov --prefix=paas --extra-repo=cf-release --extra-repo=graphite-nozzle --github-token=${GITHUB_TOKEN}
+
+.PHONY: run
+run: check-env-vars ## Unbind paas-cf of and run specified $JOB in create-bosh-cloudfoundry pipeline
```

Comment:
I think `run_job` is fine. This follows the same pattern as the `merge_pr` target. I don't think the repetition with `JOB=` is a problem.